_posts/2024-02-10_openttd-truetype-fonts.md

```diff
@@ -0,0 +1,51 @@
+---
+title: TrueType fonts for OpenTTD
+author: Zephyris
+---
+
+The pixel-art OpenTTD fonts are iconic, but were designed for 800x600 CRT monitors rather than today's 4k monsters. The pixel-perfect rendering and 2x or 4x upscaling of the old bitmap/sprite-based font gives jagged and hard-to-read text, especially as they were designed to be horizontally smoothed by CRT scanlines. How do you update a 30-year old font to make it more suitable for today?
```

Comment:
I wonder whether it was (initially) designed for 800x600. It might have been 640x480.

[This box](https://i.ebayimg.com/images/g/4-4AAOSwCCFj4sx8/s-l1600.jpg) seems to say 640x480. Similarly [Moby Games](https://www.mobygames.com/game/535/transport-tycoon-deluxe/specs/) mentions only 640x480.

I, however, do not know whether the original only supported 640x480 or whether that was just the minimum requirement.

Reply:
Running TTD in dosbox looks like 640x480... I think my memory is clouded by the windows port and TTDPatch!